x/gamm/pool-models/stableswap/amm.go

```diff
@@ -223,7 +223,13 @@ func solveCFMMBinarySearchMulti(constantFunction func(osmomath.BigDec, osmomath.
 	}
 }
 
-func spotPrice(baseReserve, quoteReserve osmomath.BigDec, remReserves []osmomath.BigDec) osmomath.BigDec {
+func (p Pool) spotPrice(baseDenom, quoteDenom string) (sdk.Dec, error) {
+	roundMode := osmomath.RoundBankers // TODO:
```

Comment:
Is TODO meant for explaining the reason for the choice of the rounding direction? I think that would be useful

Reply:
Its for us to later choose the right rounding direction. All rounding direction choices are currenetly a TODO